scripts/github-status/github-status.rb

```diff
@@ -15,6 +15,19 @@ def initialize(config = {})
   end
 
   def create_status(commit, result, description = '', target_url = '')
+    description =  @comment_prefix + case result
+      when 'success'
```

Comment:
they come from the command line arguments as strings, should I translate them to symbols then?